apps/worker/src/graph_publisher/graph.publisher.processor.service.ts

```diff
@@ -111,9 +119,29 @@ export class GraphUpdatePublisherService extends BaseConsumer {
       }
       this.logger.debug(`Tx hash: ${txHash}`);
       return txHash;
-    } catch (e) {
-      this.logger.error(`Error processing batch: ${e}`);
-      throw e;
+    } catch (error: any) {
+      this.logger.error(`Error processing batch: ${error}`);
+      throw error;
     }
   }
+
+  /**
+   * Handles low capacity by pausing the graph change notify queue.
+   * @returns A promise that resolves when the queue is paused.
+   */
+  @OnEvent('lowCapacity', { async: true, promisify: true })
+  private async handleLowCapacity(): Promise<void> {
+    this.logger.debug('Pausing graph change notify queue');
+    // await this.graphChangePublishQueue.pause();
```

Comment:
I think we will pause graph change and graph publish queue in this case

But thinking more ...